src/modules/send/ModuleSend.vue

```diff
@@ -584,6 +594,17 @@ export default {
     }, 500);
   },
   methods: {
+    verifyHexFormat() {
+      if (!this.data || isEmpty(this.data)) {
+        this.data = '0x';
+        return;
+      }
+      if (!isHexStrict(this.data)) {
```

Comment:
actually thinking more about it, this shouldn't be necessary. just let the user know to input a hex value instead of replacing the value with '0x'